locales/en-US.json

```diff
@@ -109,6 +109,7 @@
   "latestLibrary": "Latest {libraryName}",
   "libraries": "Libraries",
   "libraryEmpty": "This library is empty",
+  "libraryEmptyFilters": "No matched. Remove some filters to show more items",
```

Comment:
The first sentence feels wrong in English, I'd say something like "Nothing matched"

Reply:
I guess he wants to say "No matches", but "No matched" is indeed wrong :)

Reply:
What about "No matching items found"?

Reply:
> The first sentence feels wrong in English, I'd say something like "Nothing matched"

Yeah that's a typo

Reply:
@shadowghost The problem is that we have ``item`` repeated again in the same sentence, that's why I proposed a message without it.